.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: node_js
```

Comment:
My impression of this line is tell travis what version of node to use in its tests. I am fine just picking one version for this, something we know the dependencies that are in the package.json will run under. Not all npm deps can run in iojs or even 0.12 yet. If 0.12 works for that, then let's just stick with that.